server/data/activeLduClient.js

```diff
@@ -19,4 +19,37 @@ module.exports = {
     const { rows } = await db.query(query)
     return parseInt(rows[0].count, 10) > 0
   },
+
+  async allActiveLdusInArea(probationAreaCode) {
+    const query = {
+      text: `SELECT ldu_code FROM active_local_delivery_units WHERE probation_area_code=$1`,
+      values: [probationAreaCode],
+    }
+
+    const { rows } = await db.query(query)
+    return rows
+  },
+
+  async updateWithActiveLdu(probationAreaCode, activeLduCodes) {
+    // flush then insert all the ldus in activeLduCodes array assigning the same probationAreaCode to each one
+
+    await db.inTransaction(async client => {
+      let dbQuery
+
+      dbQuery = {
+        text: `DELETE FROM active_local_delivery_units WHERE probation_area_code = $1`,
+        values: [probationAreaCode],
+      }
+
+      await client.query(dbQuery)
+
+      activeLduCodes.forEach(async lduCode => {
+        dbQuery = {
+          text: `INSERT INTO active_local_delivery_units (probation_area_code, ldu_code) VALUES ($1, $2)`,
```

Comment:
as discussed would be better to use `format`:
e.g: 
```js
  const format = require('pg-format')
  ...
  async acknowledgeWarnings(errorIds) {
    const query = {
      text: format(`UPDATE warnings SET acknowledged = true WHERE id in (%L)`, errorIds),
    }
    const { rowCount } = await db.query(query)
    return rowCount
  },
```

Reply:
done - but discuss